colabs/JoinTables.ipynb

```diff
@@ -0,0 +1,419 @@
+            "+---------+---------+-------+\n",
+            "|tt0000001|nm0005690|     \\N|\n",
+            "|tt0000002|nm0721526|     \\N|\n",
+            "+---------+---------+-------+\n",
+            "only showing top 2 rows\n",
+            "\n",
+            "title_basics\n",
+            "+---------+---------+--------------------+--------------------+-------+---------+-------+--------------+-----------------+\n",
+            "|   tconst|titleType|        primaryTitle|       originalTitle|isAdult|startYear|endYear|runtimeMinutes|           genres|\n",
+            "+---------+---------+--------------------+--------------------+-------+---------+-------+--------------+-----------------+\n",
+            "|tt0000001|    short|          Carmencita|          Carmencita|      0|     1894|     \\N|             1|Documentary,Short|\n",
+            "|tt0000002|    short|Le clown et ses c...|Le clown et ses c...|      0|     1892|     \\N|             5|  Animation,Short|\n",
+            "+---------+---------+--------------------+--------------------+-------+---------+-------+--------------+-----------------+\n",
+            "only showing top 2 rows\n",
+            "\n",
+            "title_akas\n",
+            "+---------+--------+----------+------+--------+-----------+-------------+---------------+\n",
+            "|  titleId|ordering|     title|region|language|      types|   attributes|isOriginalTitle|\n",
+            "+---------+--------+----------+------+--------+-----------+-------------+---------------+\n",
+            "|tt0000001|       1|Карменсіта|    UA|      \\N|imdbDisplay|           \\N|              0|\n",
+            "|tt0000001|       2|Carmencita|    DE|      \\N|         \\N|literal title|              0|\n",
+            "+---------+--------+----------+------+--------+-----------+-------------+---------------+\n",
+            "only showing top 2 rows\n",
+            "\n",
+            "name_basics\n",
+            "+---------+-------------+---------+---------+--------------------+--------------------+\n",
+            "|   nconst|  primaryName|birthYear|deathYear|   primaryProfession|      knownForTitles|\n",
+            "+---------+-------------+---------+---------+--------------------+--------------------+\n",
+            "|nm0000001| Fred Astaire|     1899|     1987|soundtrack,actor,...|tt0072308,tt00319...|\n",
+            "|nm0000002|Lauren Bacall|     1924|     2014|  actress,soundtrack|tt0071877,tt01170...|\n",
+            "+---------+-------------+---------+---------+--------------------+--------------------+\n",
+            "only showing top 2 rows\n",
+            "\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Join data"
+      ],
+      "metadata": {
+        "id": "PG2BEgFuWGrB"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "data = title_basics.join(title_ratings, how=\"left\", on=\"tconst\")\n",
```

Comment:
Fajną praktyką jest przed joinowanie wyświetlić sobie liczbę rekordków i kolumn. W zasadzie przed każdym jednym joinem oraz na końcu.

Reply:
dodałem print dla wymiarów joinowanych tabel i dla końcowej tabeli

Reply:
to mało. dopisałem drugi komentarz do tego.

Reply:
dodałem printowanie wymiarów po każdym joinnie